packages/unmock-node/src/loaders/fs-service-def-loader.ts

```diff
@@ -0,0 +1,89 @@
+import debug from "debug";
+import fs from "fs";
+import path from "path";
+import { IServiceDef, IServiceDefLoader } from "unmock-core";
+
+const DEFAULT_SERVICE_SUBDIRECTORY = "__unmock__";
+
+export interface IFsServiceDefLoaderOptions {
+  servicesDir?: string;
+}
+
+const debugLog = debug("unmock:fs-service-def-loader");
+
+/**
+ * Read services from file system. Base directory is either
+ * 1. Directory injected in configuration
+ * 2. Environment variable `UNMOCK_SERVICES_DIRECTORY`
+ * 3. `${process.cwd()}/__unmock__`
```

Comment:
`process.cwd()` can be a bit tricky and might be a bit prone to errors if the user is navigating around the project workspace. I've used [app-root-path](https://www.npmjs.com/package/app-root-path). Not sure if that's the best solution, just a thought.

Reply:
True, I'll leave a TODO note as we'll need to move the resolution of the directory out of the loader soon anyway.